cmd/osm-bootstrap/osm-bootstrap.go

```diff
@@ -243,7 +242,7 @@ func applyOrUpdateCRDs(crdClient *apiclient.ApiextensionsV1Client) {
 			log.Fatal().Err(err).Msgf("Error decoding CRD file %s", file)
 		}
 
-		crd.Labels[constants.ReconcileLabel] = strconv.FormatBool(enableReconciler)
+		crd.Labels[constants.ReconcileLabel] = osmVersion
```

Comment:
maybe conditionally add the label based on enableReconciler

Reply:
Makes sense in general, but I'm wary of changing more behavior than I need to. The label wouldn't actually be used until the reconciler is turned on, so I think it's safe to keep this logic as is